src/main/scala/com/databricks/spark/sql/perf/mllib/yaml.scala

```diff
@@ -0,0 +1,184 @@
+package com.databricks.spark.sql.perf.mllib
+
+import java.util.{ArrayList => AL}
+
+import scala.concurrent.duration._
+import scala.collection.JavaConverters._
+import scala.io.Source
+
+import scala.reflect._
+import scala.reflect.runtime.universe._
+import scala.util.{Try => STry, Success, Failure}
+
+import org.yaml.snakeyaml.Yaml
+
+import com.databricks.spark.sql.perf.{ExtraMLTestParameters, MLTestParameters}
+
+
+/**
+ * The configuration information generated from reading a YAML file.
+ *
+ * @param output the output direct
+ * @param timeout
+ * @param runnableBenchmarks
+ */
+case class YamlConfig(
```

Comment:
I'm not going to review this file.  Let's make sure to check the perf results carefully to make sure they are using the correct params.  Also, can you make a note to add unit tests later on?